spec/features/feature_flags_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe "Feature flags" do
+  before do
+    allow(FeatureFlags).to receive(:static).and_return(%i[static_flag])
+    allow(FeatureFlags).to receive(:time_dependant).and_return(%i[time_dependant_flag])
+    allow(FeatureFlags).to receive(:enabled?).with(:static_flag).and_return(true)
+    allow(FeatureFlags).to receive(:enabled?).with(:time_dependant_flag).and_return(false)
+  end
+
+  scenario "I accept cookies via the banner" do
```

Comment:
is this scenario text correct?

Reply:
Not even remotely - that'll teach me to copy-paste without thinking.